A.txt

```diff
@@ -1 +1,2 @@
-Pariwesh gupta
```

Comment:
This is not correct

Reply:
Issue has been fixed.Plzzzzzzzzzzzzzz  merge it